spec/data-model/README.md

```diff
@@ -46,6 +46,15 @@ the JSON and DTD definitions are intended for interchange between systems and pr
 To that end, they relax some aspects of the data model, such as allowing
 declarations, options, and attributes to be optional rather than required properties.
 
+> [!NOTE]
+> Users relying on XML representations of messages should note that
+> XML 1.0 does not allow for the representation of all C0 control characters (U+0000-U+001F)
+> and XML 1.1 does not allow for the representation of a U+0000 NULL character.
+> These characters are allowed in MessageFormat 2 messages,
+> so systems and users relying on XML as a resource format might need to
+> apply additional restrictions on message characters
+> or provide an alternative escape mechanism for these characters.
```

Comment:
How about something like the following instead?

```suggestion
> Users relying on XML representations of messages should note that
> XML 1.0 does not allow for the representation of all C0 control characters (U+0000-U+001F).
> These characters are allowed in valid MessageFormat 2 messages.
> Thus, XML 1.1 is recommended for representing serialized MessageFormat 2 messages.
>  
> However, in MessageFormat 2 Tech Preview, valid messages may contain the U+0000 NULL character,
> which cannot be represented in XML 1.1.
> Systems and users relying on XML as a resource format might need to
> apply additional restrictions on message characters
> or provide an alternative escape mechanism for the U+0000 NULL character.
```

Reply:
(as individual contributor) 

It isn't just NULL that's a problem in XML 1.0 and, as noted, XML 1.1 isn't really a thing we can use.

Instead of mucking around with this, I think we're better off saying that some messages can't be serialized in the TP's XML data model. We can fix this _during_ the preview.

Reply:
Agreed. I was trying to suggest an improvement to the note's phrasing based on the assumption that we *could* recommend XML 1.1, and to make room for #682. At this point, however, it's best to not make any late changes hastily.

Reply:
+1

If localizable strings are stored in XML and some platform needs the controls for some reason,
they might already have a way to represent them for non MF2 messages.
If we add our own mechanism we only add to the confusion.

For example XLIFF has [`<cp>`](https://docs.oasis-open.org/xliff/xliff-core/v2.1/os/xliff-core-v2.1-os.html#cp)

Let's say my exiting XML use the same convention.
If a tool want to save "Hello <the real 0003 character> world" into such a file it has no way to know if it should escape as `<cp hex="0003"/>`, or as `\u0003` (MF2).
Because just by looking at a string there is no way to tell what API is used on it.
So the safe thing is to escape with the storage convention (`<cp hex="0003"/>`),
And we only introduce the MF2 complexity for no good reason.